public/check-ie.js

```diff
@@ -0,0 +1,16 @@
+function includes(text, target) {
+    if (text.indexOf(target) !== -1) {
+        return true;
+    }
+    return false;
+}
+
+function checkIEBrowser() {
```

Comment:
```suggestion
function isIEBrowser() {
```

"ie browser 면 무엇을 해라" 라는 플로우로 읽히도록, 함수명을 이렇게 바꿔보면 어떨까요??